pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```diff
@@ -38,30 +56,20 @@ stdenv.mkDerivation (finalAttrs: {
     s
     servant
     shell-split-string
-  ]);
-
-  strictDeps = true;
-
-  buildPhase = ''
-    runHook preBuild
+  ];
 
-    emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
-
-    runHook postBuild
-  '';
+  doCheck = true;
```

Comment:
If you check the log, you will find that test is not actually run.  There are two causes:
1. elisp builders do not work well with the default `checkPahse` of `stdenv`
2. there seems to be issues between the default stdenv checkPhase and cask `Makefile`, meaning test is not run even before this PR.

https://hydra.nixos.org/build/269572907

> Running phase: checkPhase
> no Makefile or custom checkPhase, doing nothing


I will make a PR to fix the first cause.

A workaround for the second cause is to set `checkTarget = "test"`.  However, the test fails.  But that is a separate issue.

Reply:
> I will make a PR to fix the first cause.

https://github.com/NixOS/nixpkgs/pull/338604

Reply:
I will delete this, then.